core/server/services/settings/index.js

```diff
@@ -4,16 +4,78 @@
  */
 
 var SettingsModel = require('../../models/settings').Settings,
-    SettingsCache = require('./cache');
+    SettingsCache = require('./cache'),
+    SettingsLoader = require('./loader'),
+    _ = require('lodash'),
+    common = require('../../lib/common');
+    // SettingsFiles = require('./ensure-settings');
 
 module.exports = {
     init: function init() {
+        // TODO: the ensure settings fn up before the settings cache
+        // return SettingsFiles(this.knownSettings()).then(() => {
+
         // Update the defaults
         return SettingsModel.populateDefaults()
             .then(function (settingsCollection) {
                 // Initialise the cache with the result
                 // This will bind to events for further updates
                 SettingsCache.init(settingsCollection);
             });
+        // });
+    },
+
+    // TODO: is this the right place for it? We need a global place where
+    // we can add additional files that are supported.
+    // This can also be a config value for example.
+    knownSettings: function knownSettings() {
+        return ['routes'];
```

Comment:
This won't work.
The file is called `default-routes.yml`. If you start Ghost and enable `ensure-settings`, it will immedetially throw an error that `routes.yaml` does not exist in the core folder.